google/cloud/spanner/integration_tests/database_admin_integration_test.cc

```diff
@@ -207,6 +208,12 @@ TEST_F(DatabaseAdminClientTest, DatabaseBasicCRUD) {
           FirstName  STRING(1024),
           LastName   STRING(1024),
           SingerInfo BYTES(MAX)
+      )""");
+  if (!emulator_) {
```

Comment:
Should we just create a separate test for JSON?  All these `if (!emulator_)` things sprinkled over the test are hard to grok.

Reply:
At least for the time being, I think I prefer this over duplication.  But part of the "grokness" issue depends on how long these conditionals last.  I'm still hoping that the emulator will support some of these features soon, but who knows.